frontend/components/organiser/event/custom-event-links/CustomEventLinksTable.tsx

```diff
@@ -0,0 +1,380 @@
+import { CustomEventLink, CustomEventLinkType, EMPTY_CUSTOM_EVENT_LINK } from "@/interfaces/CustomLinkTypes";
+import { EventData } from "@/interfaces/EventTypes";
+import { RecurrenceTemplate } from "@/interfaces/RecurringEventTypes";
+import { UserData } from "@/interfaces/UserTypes";
+import {
+  deleteCustomEventLink,
+  saveCustomEventLink,
+} from "@/services/src/events/customEventLinks/customEventLinksService";
+import { DocumentDuplicateIcon, PencilIcon, TrashIcon } from "@heroicons/react/24/outline";
+import { Button, Chip, IconButton, Input, Option, Select, Tooltip, Typography } from "@material-tailwind/react";
+import { useEffect, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+
+const TABLE_HEAD = ["Custom Link Name", "Custom Link", "Type", "Reference", ""];
```

Comment:
Could be good to give some info (e.g on hover) for what purpose each of these fields serve. Esp for peeps who are learning custom event links for the first time - we can do this in a follow up PR